.github/pull_request_template.md

```diff
@@ -1,6 +1,6 @@
 ## Ticket
 
-https://wicmtdp.atlassian.net/browse/{TICKET_ID}
+{TICKET_LINK}
```

Comment:
I probably want to make some obvious `REPLACE_ME` in a few places in the template.

Reply:
Good call